scripts/run_interop_tests.sh

```diff
@@ -47,6 +47,14 @@ docker rm -f "$pid1"
 docker rm -f "$pid2"
 
 
+#
+# Run interop tests against grpc-web java connector code
+#
+docker-compose build java-interop-server
+pid3=$(docker run -d --network=host grpcweb/java-interop-server)
+docker run --network=host --rm grpcweb/prereqs /bin/bash /github/grpc-web/scripts/docker-run-interop-tests.sh
```

Comment:
Nit: can we break this long line like L43-44 above? Just to be consistent.

Thanks

Reply:
done. (made a shell function to call tests)